### ISSUE TITLE
Modpack version selector for platforms that supports version selection (Like Technic)

### ISSUE THREAD

**Issue by Hexality** [MultiMC/Launcher#4649]
### Role

I use MultiMC to play technic modpacks and some other platforms.

### Suggestion

The ability to chose a specific version of the modpack to download.

### Benefit

So I can select a specific version of the modpack that works better with my maps.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

Basically what I want to suggest is a dropdown menu to select a specific version of a modpack to use instead of it always downloading the latest recommended version.
How it is in technic:
![image](https://user-images.githubusercontent.com/17398632/164053188-1456b6d4-c165-44ea-99fe-e303988033ae.png)

Example of how it could be on MultiMC:
![image](https://user-images.githubusercontent.com/17398632/164054404-6a3b05aa-26a1-4c41-9671-ae030d43bb8a.png)

**Comment by phit** [comment]
Duplicate of #3516
